.github/workflows/deployments.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: |
           pip install --no-dependencies tests/resources/mlflow-test-plugin
           pip install .[gateway] \
-            pytest pytest-timeout pytest-asyncio httpx psutil sentence-transformers transformers
+            pytest pytest-timeout pytest-asyncio httpx psutil sentence-transformers transformers databricks-sdk
```

Comment:
```suggestion
            pytest pytest-timeout pytest-asyncio httpx psutil sentence-transformers transformers
```

Reply:
After removing this , error occurs: https://github.com/mlflow/mlflow/actions/runs/9189044683/job/25270206515?pr=12011#step:6:503